docs/BreakingChanges/109 Building an Entity Framework edmx with Visual Studio can fail with error MSB4062.md

```diff
@@ -0,0 +1,35 @@
+## 109: Building an Entity Framework edmx with Visual Studio 2013 can fail with error MSB4062 if using the EntityDeploySplit or EntityClean tasks
```

Comment:
{
  "Id": "109",
  "Title": "Building an Entity Framework edmx with Visual Studio 2013 can fail with error MSB4062 if using the EntityDeploySplit or EntityClean tasks",
  "Details": "MSBuild 12.0 tools (included in Visual Studio 2013) changed msbuild file locations, causing older Entity Framework targets files to be invalid. The result is that `EntityDeploySplit` and `EntityClean` tasks fail because they are unable to find `Microsoft.Data.Entity.Build.Tasks.dll`. Note that this break is because of a toolset (msbuild/VS) change, not because of a .NET Framework change. It will only occur when upgrading developer tools, not when merely upgrading the .NET Framework.",
  "Suggestion": "Entity Framework targets files are fixed to work with the new msbuild layout beginning in the .NET Framework 4.6. Upgrading to that version of the Framework will fix this issue. Alternatively, [this](http://stackoverflow.com/a/24249247/131944) workaround can be used to patch the targets files directly.",
  "Link": "http://stackoverflow.com/questions/20400054/entitydeploysplit-error-microsoft-data-entity-build-tasks-dll-missing/24249247#24249247",
  "Markdown": null,
  "ApplicableApis": [
    "Not detectable via API analysis"
  ],
  "Related": null,
  "VersionBroken": "4.5.1",
  "VersionFixed": "4.6",
  "IsBuildTime": true,
  "IsQuirked": false,
  "IsSourceAnalyzerAvailable": false,
  "BugLink": "http://vstfdevdiv:8080/DevDiv2/DevDiv/_workitems/edit/684684",
  "Notes": "This isn't something we could very accurately detect with ApiPort, but it may be possible to catch it with a non-code analyzer\nSource analyzer status: Pri 2",
  "IsRetargeting": true,
  "ImpactScope": 1
}

**Parsed APIs:**
- [ ] Not detectable via API analysis